XenAdmin/Wizards/GenericPages/SelectMultipleVMDestinationPage.cs

```diff
@@ -404,12 +405,20 @@ private void PopulateDataGridView(IEnableableXenObjectComboBoxItem selectedItem)
                             }
                         }
 
+                        var items = new List<DelayLoadingOptionComboBoxItem>();
+
                         foreach (var host in Connection.Cache.Hosts)
                         {
                             var item = new DelayLoadingOptionComboBoxItem(host, homeserverFilters);
                             item.LoadAndWait();
-                            cb.Items.Add(item);
+                            items.Add(item);
+                        }
+                        items.Sort(new DelayLoadingOptionComboboxItemCompare());

```

Comment:
Can we use a natural compare here please? Currently the order is 1,10,11, etc - we have a natural comparer that handles numbers :) See `StringUtility.NaturalCompare`

Reply:
Actually, whenever we sort the hosts in XenCenter we should use the default sort for the Host class, which sorts in the following order:
 * Masters first
 * Then connected slaves
 * Then disconnected servers
 Within each group, in NaturalCompare order

Reply:
It worked. And sorting result looks good in this way. 👍